index.html

```diff
@@ -77,10 +64,9 @@ <h1>Welcome to Malcolm’s CSD120 Website!</h1>
           That being said, a lot of work is becoming remote in the computer
           realm so I could also remain here in Sault Ste Marie.
         </p>
-      </div>
 
-      <div class = "tab2">
```

Comment:
No need to do all this wrapping you did in the lab.  Instead, keep your HTML clean and use appropriately specific selectors